<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>391</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>137</width>
      <height>585</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="openImgBtn">
       <property name="text">
        <string>Open Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveImgBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetImageBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="displayImageBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Display Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="grayBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Turn to Gray</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mirVerBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mirror Vertical</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mirHorBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mirror Horizontal</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quantBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Quantize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adjustBrightnessBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Adjust Brightness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adjustContrastBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Adjust Contrast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="negativeBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Negative</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotatePlusBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rotate +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotateMinusBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rotate -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="convolutionBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Convolution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Zoom out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomInBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Zoom in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="histogramBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="equalizeHistogramBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Equalize Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="matchHistogramBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Match Histogram</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
